2020/02/08 05:31:27 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 05:35:00 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 05:45:02 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 05:49:24 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 05:55:08 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 06:02:48 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 09:26:03 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 09:34:57 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 09:35:27 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:16:27 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:19:11 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:20:26 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:35:30 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:35:39 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:39:18 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 11:02:04 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 11:02:12 [crit] 7#7: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.29.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/08 11:02:18 [crit] 7#7: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.29.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/08 11:12:48 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 11:13:15 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 11:16:17 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 15:09:18 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 15:47:35 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 15:52:24 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 15:53:03 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 15:53:14 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost:3000"
2020/02/08 15:53:14 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/favicon.ico", host: "localhost:3000", referrer: "http://localhost:3000/"
2020/02/08 15:53:22 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost:3000"
2020/02/08 15:53:22 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/favicon.ico", host: "localhost:3000", referrer: "http://localhost:3000/"
2020/02/08 15:53:40 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost:3000"
2020/02/08 15:53:40 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/favicon.ico", host: "localhost:3000", referrer: "http://localhost:3000/"
2020/02/08 16:05:23 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 16:15:06 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 16:16:58 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 16:23:44 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 16:24:28 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 16:33:23 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 16:33:45 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 16:33:59 [crit] 8#8: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.32.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost:3000"
2020/02/08 16:33:59 [crit] 8#8: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.32.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/favicon.ico", host: "localhost:3000", referrer: "http://localhost:3000/"
2020/02/08 16:34:25 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 16:51:53 [crit] 6#6: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost:3000"
2020/02/08 16:51:54 [crit] 6#6: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/favicon.ico", host: "localhost:3000", referrer: "http://localhost:3000/"
2020/02/11 08:32:56 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 08:47:49 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 08:48:14 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 08:50:03 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 08:50:45 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 08:52:22 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 08:53:40 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 09:35:27 [crit] 8#8: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 09:36:09 [crit] 8#8: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (13: Permission denied) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 09:36:11 [crit] 8#8: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (13: Permission denied) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 09:37:35 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 09:37:36 [crit] 7#7: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (13: Permission denied) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 09:37:41 [crit] 7#7: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (13: Permission denied) while connecting to upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 10:09:24 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 10:12:28 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 10:14:32 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.24.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 11:50:45 [crit] 6#6: *4 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.24.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 13:27:31 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 13:27:57 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.25.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 13:33:22 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 13:33:36 [crit] 6#6: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 13:34:17 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/11 13:34:35 [crit] 7#7: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/11 13:35:00 [crit] 7#7: *2 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/12 03:56:31 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/12 03:57:20 [crit] 7#7: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/12 13:10:27 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
